### PR TITLE
MINOR : Add s390x support for Travis

### PR DESCRIPTION
**Pull request by sangitanalkar**
As Travis CI [officially supports](https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z) s390x builds, adding support for building Apache storm for s390x in travis.

### CODE REVIEW

**Comment by sangitanalkar** [comment]
Hi, any updates on the PR if the changes can be merged?

File: .travis.yml [hunk 1]
```diff
@@ -46,9 +52,12 @@ before_install:
   - export MVN_HOME=$HOME/apache-maven-3.6.1
   - if [ ! -d $MVN_HOME/bin ]; then wget https://archive.apache.org/dist/maven/maven-3/3.6.1/binaries/apache-maven-3.6.1-bin.tar.gz -P $HOME; tar xzvf $HOME/apache-maven-3.6.1-bin.tar.gz -C $HOME; fi
   - export PATH=$MVN_HOME/bin:$PATH
+
 install: /bin/bash ./dev-tools/travis/travis-install.sh `pwd`
 script:
-  - /bin/bash ./dev-tools/travis/travis-script.sh `pwd` $MODULES
+  - if [[ $(uname -m) != 's390x' ]]; then
```
Comment (kishorvpatil) [inline .travis.yml]: Why not build all modules on s390x as well?

Comment (sangitanalkar) [reply .travis.yml]: Hi @kishorvpatil , 
"travis-script.sh" contains tests for each of the MODULES and for some of the tests rocksdb is required. Currently there is no support for rocksdb for s390x. Found a PR raised to add s390x support to rocksdb ([link](https://github.com/facebook/rocksdb/pull/6168)) , waiting for it to be merged.

**Comment by kishorvpatil** [comment]
Can you please prefix  the title and commit with MINOR to suggest this is a minor change.

**Comment by sangitanalkar** [comment]
Hi @kishorvpatil , currently there is no support for rocksdb for s390x which is required to run the MODULES. 
Can you please let us know if we can go ahead with this PR?

File: .travis.yml (hunk 1 above)
Comment (Ethanlm) [reply .travis.yml]: Travis build has been failing. Might be related to this

Comment (sangitanalkar) [reply .travis.yml]: @Ethanlm all builds have passed now.

**Comment by Ethanlm** [comment]
Thanks for the contribution.
My question on this is: Is anyone using s390x for Storm? If anything fails on s390x, somebody needs to spend time debugging. It would not be straightforward to me since I use `amd64` only. I wonder if it's worth doing it. Thanks

**Comment by sangitanalkar** [comment]
> My question on this is: Is anyone using s390x for Storm? If anything fails on s390x, somebody needs to spend time debugging. It would not be straightforward to me since I use `amd64` only. I wonder if it's worth doing it. Thanks

Hi @Ethanlm . Yes, we already build and maintain Apache Storm on s390x. Our build instructions are published [here](https://github.com/linux-on-ibm-z/docs/wiki/Building-Apache-Storm) .

**Comment by Ethanlm** [comment]
@sangitanalkar  Thanks. I merged this to master.

**Pull request merged**